src/pages/console/tutorial/start.mdx

```diff
@@ -54,13 +35,14 @@ If you already have an existing backend environment, you can enable Amplify Stud
 
 ### To get started from an existing Amplify project
 
+<Callout>
+
+To create a Amplify application using Command Line Interface (CLI) refer to [Amplify CLI](/cli/start/install). 
```

Comment:
```suggestion
To create a Amplify project using Command Line Interface (CLI) refer to [Amplify CLI](/cli/start/install). 
```

see https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#2-amplify-project-not-amplify-app